File: TestData/CSToVBCharacterization/ConvertSingleProject/ConsoleApp2/ユニコードのプログラム.vb
Imports System
Imports System.Diagnostics
Imports System.Linq

Namespace ConsoleApp2
    Public Class ユニコードのプログラム
        Public Shared Sub Main()
            For Each process In Process.GetProcesses().Where(Function(p) String.IsNullOrEmpty(p.MainWindowTitle)).Take(1)
                Console.WriteLine(-1)
            Next
        End Sub
    End Class
End Namespace

File: TestData/CharacterizationTestSolution/ConsoleApp2/ユニコードのプログラム.cs
using System;
using System.Diagnostics;
using System.Linq;

namespace ConsoleApp2
{
    public class ユニコードのプログラム
    {
        public static void Main()
        {
            foreach (var process in Process.GetProcesses().Where(p => String.IsNullOrEmpty(p.MainWindowTitle)).Take(1))
            {
                // Here's a comment
                Console.WriteLine(-1);
            }
        }

    }
}
